src/main/java/com/github/kokorin/jaffree/process/BaseStdReader.java

```diff
@@ -82,6 +86,7 @@ public T read(final InputStream stdOut) {
                     case ERROR:
                     case FATAL:
                     case PANIC:
+                        errorLogMessages.add(logMessage);
```

Comment:
Is it possible to move it to `case QUITE:`?

Reply:
Sure, but is QUITE really a valid error log level?